src/api/python/cvc5.pxi

```diff
@@ -501,9 +508,19 @@ cdef class Solver:
         sort.csort = self.csolver.mkDatatypeSort(dtypedecl.cdd)
         return sort
 
-    def mkDatatypeSorts(self, list dtypedecls, unresolvedSorts):
+    def mkDatatypeSorts(self, *args): 
```

Comment:
Since the only two options are to have one or two arguments, it would be better to use a default argument. Something like:

```
def mkDatatypeSorts(self, list dtypedecls, set unresolvedSorts = None)
```

and then you can check whether `unresolvedSorts is None`.

Reply:
Yes, that's better, thanks!
Fixed here: 723ecc0da26e5ecfa26d4e49a7b943acedd6aee5

While fixing this, I found the need to distinguish between cpp sets (now called `c_set`s) and python sets. Done here: https://github.com/cvc5/cvc5/pull/6640/commits/4c4b697e9564c454f58acec1c7de98da813feaf4